net/nimble/include/nimble/hci_common.h

```diff
@@ -458,19 +458,49 @@ extern "C" {
 
 /* --- LE set extended advertising parameters (OCF 0x0036) */
 #define BLE_HCI_LE_SET_EXT_ADV_PARAM_LEN            (25)
+#define BLE_HCI_LE_SET_EXT_ADV_PROP_CONNECTABLE     (0x0001)
+#define BLE_HCI_LE_SET_EXT_ADV_PROP_SCANNABLE       (0x0002)
+#define BLE_HCI_LE_SET_EXT_ADV_PROP_DIRECTED        (0x0004)
+#define BLE_HCI_LE_SET_EXT_ADV_PROP_HD_DIRECTED     (0x0008)
+#define BLE_HCI_LE_SET_EXT_ADV_PROP_LEGACY          (0x0010)
+#define BLE_HCI_LE_SET_EXT_ADV_PROP_ANON_ADV        (0x0020)
+#define BLE_HCI_LE_SET_EXT_ADV_PROP_INC_TX_PWR      (0x0040)
+#define BLE_HCI_LE_SET_EXT_ADV_PROP_MASK            (0x7F)
+
+#define BLE_HCI_LE_SET_EXT_ADV_PROP_LEGACY_IND      (0x0013)
```

Comment:
only first 5 props match what is set also in extended advertising event, but those are for separate command and (there is also anon adv and tx power which are not present in event), also mask size is different,

I'd leave it as is